triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/ServiceTest.scala

```diff
@@ -167,15 +168,16 @@ class ServiceTest extends AsyncFlatSpec with Eventually with Matchers with Postg
     } yield triggerIds
   }
 
-  def assertTriggerIds(uri: Uri, party: String, expectedTriggerIds: Vector[String]) = {
-    Future {
-      eventually {
-        val actualTriggerIds = Await.result(for {
-          resp <- listTriggers(uri, party)
-          result <- parseTriggerIds(resp)
-        } yield result, Duration.Inf)
-        assert(actualTriggerIds == expectedTriggerIds)
-      }
```

Comment:
It may be useful to add the more specific helper which is commonly used, i.e. `assertTriggerIdsEqual`.